Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs

```diff
@@ -236,19 +236,8 @@ private void AddSmiteVerbs(GetVerbsEvent<Verb> args)
                     if (damageToDeal <= 0)
                         damageToDeal = 100; // murder time.
 
-                    if (_inventorySystem.TryGetSlots(args.Target, out var slotDefinitions))
-                    {
-                        foreach (var slot in slotDefinitions)
-                        {
-                            if (!_inventorySystem.TryGetSlotEntity(args.Target, slot.Name, out var slotEnt))
-                                continue;
-
-                            RemComp<InsulatedComponent>(slotEnt.Value); // Fry the gloves.
-                        }
-                    }
-
                     _electrocutionSystem.TryDoElectrocution(args.Target, null, damageToDeal,
-                        TimeSpan.FromSeconds(30), true);
```

Comment:
Keep the insul-destruction, it's funny. Or add a second smite (with new icon) that doesn't destroy insuls.